packages/terra-dropdown-button/src/terra-dev-site/test/dropdown-button/BlockDropdownButton.test.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import DropdownButton, { Button } from '../../../DropdownButton';
+
+export default () => (
+  <DropdownButton
+    label="Primary option"
+    onClick={() => { console.log('primary option'); }}
+    isBlock
+  >
+    <Button label="1st Option" onClick={() => { console.log('hi'); }} />
```

Comment:
Let's update this example to write content to the page instead of a console.log.

Reply:
I don't know if the test pages (aside from the callback testing page) need to have visible callbacks like this.